.travis.yml

```diff
@@ -13,8 +13,6 @@ cache:
 
 env:
   matrix:
-    # test tensorflow keras backend
-    - KERAS_BACKEND="tensorflow" COVERAGE="true"
```

Comment:
We need to keep this line, you can just remove the `KERAS_BACKEND="tensorflow"` part of it.  You could have these too lines look like this:

```
# run tests, submit coverage info
- COVERAGE="true"
```